### ISSUE TITLE
frida-gum.d.t - Error TS1036: Statements are not allowed in ambient contexts.

### ISSUE THREAD

**Issue by TheKalin** [frida/frida-gum#283]
Hello, it looks like you have a typo in your code there is en extra ; char at the end of namespace declaration. 

https://github.com/frida/frida-gum/blob/master/bindings/gumjs/types/frida-gum/frida-gum.d.ts#L162

```
r2pm -i r2frida
Already up to date.
Install Done For r2frida
[ "`readlink ext/frida`" = frida-linux-11.0.1 ] || \
	(cd ext && rm -f frida ; ln -fs frida-linux-11.0.1 frida)
make io_frida.so
make[1]: Entering directory '/root/.local/share/radare2/r2pm/git/r2frida'
npm run build

> r2frida-agent@2.5.1 build /root/.local/share/radare2/r2pm/git/r2frida
> frida-compile src/agent -o src/_agent.js

{ [TypeScript error: node_modules/frida-gum-types/frida-gum/frida-gum.d.ts(162,2): Error TS1036: Statements are not allowed in ambient contexts.]

```

**Comment by oleavr** [comment]
Fix released in `frida-gum-types@1.1.2` 🎉

**Comment by TheKalin** [comment]
Yay. 
BTW. I have found another issue with the code. During build from source, output is simmilar to this
```
 build/frida-meson-env-linux-x86_64.rc && /opt/frida/releng/ninja-linux-x86_64 -C build/tmp-linux-x86/frida-core src/frida-helper lib/agent/frida-agent.so
ninja: Entering directory `build/tmp-linux-x86/frida-core'
[3/22] Compiling Vala source ../../../frida-core/src/linux/frida-hel.../linux/frida-helper-types.vala lib/interfaces/frida-interfaces.vapi.
FAILED: src/frida-helper-service@sta/frida-helper-service.c src/frida-helper-service@sta/frida-helper-types.c src/frida-helper-service.h src/frida-helper-service.vapi 
/opt/frida/build/toolchain-linux-x86_64/bin/valac-0.42 --vapidir="/opt/frida/build/frida-linux-x86/share/vala/vapi" --vapidir="/opt/frida/build/sdk-linux-x86/share/vala/vapi" '-C' '--define=LINUX' '--pkg' 'frida-gum-1.0' '--pkg' 'gee-0.8' '--pkg' 'gio-unix-2.0' '--pkg' 'gio-2.0' '-d' 'src/frida-helper-service@sta' '--library=frida-helper-service' '-H' 'src/frida-helper-service.h' '--use-header' '--vapi' '../frida-helper-service.vapi' '--pkg=posix' ../../../frida-core/src/linux/frida-helper-service.vala ../../../frida-core/src/linux/frida-helper-types.vala lib/interfaces/frida-interfaces.vapi
../../../frida-core/src/linux/frida-helper-service.vala:76.5-76.29: error: The name `connect' does not exist in the context of `bool'
				connection.closed.connect (on_connection_closed);
				^^^^^^^^^^^^^^^^^^^^^^^^^
../../../frida-core/src/linux/frida-helper-service.vala:57.5-57.32: error: The name `disconnect' does not exist in the context of `bool'
				connection.closed.disconnect (on_connection_closed);
				^^^^^^^^^^^^^^^^^^^^^^^^^^^^
../../../frida-core/src/linux/frida-helper-service.vala:147.35-147.47: warning: Posix.SIGKILL has been deprecated since vala-0.40. Use Posix.Signal.KILL
Compilation failed: 2 error(s), 1 warning(s)
[4/22] Compiling Vala source ../../../frida-core/lib/agent/agent.val....vala lib/interfaces/frida-interfaces.vapi lib/pipe/frida-pipe.vapi.
FAILED: lib/agent/frida-agent@sha/agent.c lib/agent/frida-agent@sha/script-engine.c lib/agent/frida-agent.h lib/agent/frida-agent.vapi 
/opt/frida/build/toolchain-linux-x86_64/bin/valac-0.42 --vapidir="/opt/frida/build/frida-linux-x86/share/vala/vapi" --vapidir="/opt/frida/build/sdk-linux-x86/share/vala/vapi" '-C' '--define=LINUX' '--pkg' 'frida-gumjs-1.0' '--pkg' 'frida-gum-1.0' '--pkg' 'gee-0.8' '--pkg' 'gio-2.0' '-d' 'lib/agent/frida-agent@sha' '--library=frida-agent' '-H' 'lib/agent/frida-agent.h' '--use-header' '--vapi' '../frida-agent.vapi' ../../../frida-core/lib/agent/agent.vala ../../../frida-core/lib/agent/script-engine.vala lib/interfaces/frida-interfaces.vapi lib/pipe/frida-pipe.vapi
../../../frida-core/lib/agent/agent.vala:11.8-11.31: error: The name `try_get_range' does not exist in the context of `Gum.Thread'
			if (Gum.Thread.try_get_range (out stack))
			    ^^^^^^^^^^^^^^^^^^^^^^^^
../../../frida-core/lib/agent/agent.vala:169.5-169.32: error: The name `disconnect' does not exist in the context of `bool'
				connection.closed.disconnect (on_connection_closed);
				^^^^^^^^^^^^^^^^^^^^^^^^^^^^
../../../frida-core/lib/agent/agent.vala:156.4-156.28: error: The name `connect' does not exist in the context of `bool'
			connection.closed.connect (on_connection_closed);
			^^^^^^^^^^^^^^^^^^^^^^^^^
Compilation failed: 3 error(s), 0 warning(s)
[7/22] Compiling C object 'lib/interfaces/frida-interfaces@sta/lib_interfaces_frida-interfaces@sta_session.c.o'.
ninja: build stopped: subcommand failed.
Makefile.linux.mk:363: recipe for target 'build/tmp-linux-x86/frida-core/.frida-helper-and-agent-stamp' failed
make[1]: *** [build/tmp-linux-x86/frida-core/.frida-helper-and-agent-stamp] Error 1
make[1]: Leaving directory '/opt/frida'
Makefile:2: recipe for target 'python-64' failed

```

It looks like some kind of inconsistency in the .vapi files .
